components/security-mgt/org.wso2.carbon.security.mgt.ui/src/main/resources/web/securityconfig/view-scenario-detail_ajaxprocessor.jsp

```diff
@@ -19,8 +19,14 @@
 <%
     String scenarioIdNumber = "";
     String scenarioId = request.getParameter("scenarioId");
+    try {
+        Integer.parseInt(scenarioId);
+    } catch (Exception e) {
+        //scenarioId should be an integer. Otherwise set to null.
+        scenarioId = null;
+    }
     String scenarioSummary = request.getParameter("scenarioSummary");
-    if(scenarioId != null && !scenarioId.equals("")){
+    if (scenarioId != null && !scenarioId.equals("")) {
```

Comment:
Since we have already checked whether scenarioId is a valid Integer, checking for `!scenarioId.equals("")` is not required.